src/features/vote/PastRequests.tsx

```diff
@@ -70,6 +70,7 @@ const PastRequests: FC<Props> = ({
         Promise.all(pr).then((res) => {
           setPastRequests(!showAll ? res.slice(0, 10) : res);
         });
```

Comment:
this needs a catch

Reply:
Doesn't even need to be a promise >.< fixed.